From 8803636bf4e0238d0f23bedebdcb4d1e6283ffb7 Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Mon, 26 Apr 2021 17:09:51 +0200
Subject: [PATCH] Make dependency on KI18n explicit

It already pulled in by KParts, but we better not rely on that
---
 src/kontact_plugin/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/kontact_plugin/CMakeLists.txt b/src/kontact_plugin/CMakeLists.txt
index dec4b8b..f1d56ec 100644
--- a/src/kontact_plugin/CMakeLists.txt
+++ b/src/kontact_plugin/CMakeLists.txt
@@ -13,7 +13,7 @@ set(kontact_kjots_PART_SRCS
 )
 
 add_library(kontact_kjotsplugin MODULE ${kontact_kjots_PART_SRCS})
-target_link_libraries(kontact_kjotsplugin KF5::KontactInterface)
+target_link_libraries(kontact_kjotsplugin KF5::I18n KF5::KontactInterface)
 kcoreaddons_desktop_to_json(kontact_kjotsplugin kjotsplugin.desktop)
 
 install(TARGETS kontact_kjotsplugin DESTINATION ${KDE_INSTALL_PLUGINDIR}/kontact5)
-- 
GitLab

